client/lib/wpcom-undocumented/lib/me.js

```diff
@@ -252,10 +252,14 @@ UndocumentedMe.prototype.devices = function( callback ) {
 	return this.wpcom.req.get( args, callback );
 };
 
-UndocumentedMe.prototype.sendVerificationEmail = function( callback ) {
+UndocumentedMe.prototype.sendVerificationEmail = function( userId, nonce, callback ) {
```

Comment:
It would be a good idea to add a docblock comment here since we have optional parameters.